ang/civicase/case/directives/case-overview.directive.js

```diff
@@ -39,16 +39,21 @@
   /**
    * Controller for civicaseCaseOverview
    *
-   * @param {Object} $scope
+   * @param crmApi
+   * @param BrowserCache
+   * @param CaseStatus
+   * @param CaseTypeCategory
+   * @param CaseType
```

Comment:
this does not throw a lint error because of missing type and description?

Reply:
When I opened the file, I saw linting errors, not sure why husky did not complain. I swear I did not do `--no-verify` :/

Reply:
it's ok. Someone will catch it later. But let's be vigilant about this.